src/server/Document/Renderers/LiteRenderer.tsx

```diff
@@ -23,12 +23,13 @@ export default function LitePageRenderer({
         {title}
         {helmetMetaTags}
         {helmetLinkTags}
-        {helmetScriptTags}
         <style dangerouslySetInnerHTML={{ __html: styles }} />
+        {/* IMPORTANT: ComponentTracking MUST come before helmetScriptTags due to synchronous calls from helmetScriptTags to functions within ComponentTracking */}
         <ComponentTracking
           enableStaticClickTrackingOnOperaMiniOnly={false}
           trackComponentViews
         />
+        {helmetScriptTags}
```

Comment:
Will this have any side effects?

Reply:
Not sure... we haven't seen any - and all of the E2E tests seem to be passing OK... Can you think of anything weird we should be looking out for?

Reply:
Nothing of the top of my head, it should be fine as none of the helmet scripts are relying on tracking afaik

Reply:
Yeah we couldn't see any errors, but we can monitor the tests in the pipeline after merging just in case it catches something we're not aware of on local.